### PR TITLE
ui: Amends to existing topology notice/banner texts

### PR DESCRIPTION
**Pull request by johncowen**
### Description
We've decided to make some text and logic amends to the topology notices/banners. This PR addresses the text changes only, partly because the logic changes are likely to affect line styling (dotted vs non-dotted) and other things in the topology visualisation itself.

Unchanged Texts:

- No Dependencies
- Limited Access

Additional texts ready for usage with to come logic.

- No intentions

All other texts apart from the above have been changed, please let me know if the unchanged texts do need changing.

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [x] not a security concern

### CODE REVIEW

**Comment by LevelbossMike** [comment]
Taking myself off review as I'm not aware what the exact texts should be.

File: ui/packages/consul-ui/translations/routes/en-us.yaml [hunk 1]
```diff
@@ -213,12 +213,19 @@ dc:
                 <a href="{CONSUL_DOCS_URL}/connect/registration/service-registration#upstream-configuration-reference" target="_blank" rel="noopener noreferrer">Read the documentation</a>
               </p>
           acls-disabled:
-            header: Enable ACLs
-            body: This connect-native service may have dependencies, but Consul isn't aware of them when ACLs are disabled. Enable ACLs to make this view more useful.
+            header: All service-to-service connections are allowed
```
Comment (jkirschner-hashicorp) [inline ui/packages/consul-ui/translations/routes/en-us.yaml:216]: @jnwright , @johncowen : Should this be consistent with the other "Restrict which services can connect" headers?

```suggestion
            header: Restrict which services can connect
```

Comment (jnwright) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:216]: Hm, I'm not sure. The title you and I chose during our sessions is the one John wrote. I remember us wanting to differentiate it for a reason, but do you think it should be consistent?

Comment (jkirschner-hashicorp) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:216]: I vaguely recall such a conversation, but not what the outcome/decision was!

The two messages are basically the same, just with a different focus. "Restrict which services can connect" focuses on what the user should do. "All service-to-service connections are allowed" focuses on communicating current behavior. I'm not sure why we'd frame the same message in two different ways depending on whether ACLs are disabled vs enabled with default allow. ... Maybe we did have a reason when we last spoke about it, but I can't remember if it we did!

Comment (jnwright) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:216]: I'm fine with changing it to be the same title ¯\_(ツ)_/¯

Comment (johncowen) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:216]: What was the outcome here? Shall I go ahead and commit the suggestion or leave as is?

Comment (jnwright) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:216]: yeah, I say we go ahead and change it! And that'll wrap this up 😎

Comment (johncowen) [reply ui/packages/consul-ui/translations/routes/en-us.yaml]: Perfect, k thats done, just needs a 👍 /approval on this one then and we can merge

**Review by WenInCode — APPROVED** [review]

**Review by jnwright — APPROVED** [review]

**Pull request merged**